docs/3.api/5.kit/11.nitro.md

```diff
@@ -358,3 +358,40 @@ export default defineNuxtModule({
   }
 })
 ```
+
+## `addServerRoutesDir`
+
+Add a directory to be scanned for server routes by Nitro.
+
+### Type
+
+```ts
+function function addServerRoutesDir (dirs: string | string[], opts: { prepend?: boolean }): void
+```
+
+### Parameters
+
+#### `dirs`
+
+**Type**: `string | string[]`
+
+**Required**: `true`
+
+A directory or an array of directories to register to be scanned for API routes by Nitro
```

Comment:
To avoid confusion with (unprefixed) nitro routes, it would be better to be named AddAPIRoutesDir / AddServerAPIRoutesDir

Reply:
what about `addServerScanDir` ? I think nitro will scan everything just like the `server` directory, meaning `/api`, `/routes` and `/utils` are all scanned right ?

Reply:
Yeas, considering how it is implemented now, this utility adds scan dir. Would be also nice at the same time think of a consistent name for this functionality as we were also planning to introduce a nuxt kit utility that adds layer-like dirs that have `pages`, etc.)